backend/src/main/java/com/empire/World.java

```diff
@@ -2406,7 +2449,6 @@ void advanceDate() {
 
 	private void addPopulation(Region r, double amount) {
 		r.population += amount;
-		int rid = regions.indexOf(r);
```

Comment:
This comment is not really related to the change made, but this method really looks like it should belong to the Region class.

Reply:
Yeah.